src/EditorFeatures/Test2/IntelliSense/SignatureHelpControllerTests.vb

```diff
@@ -98,6 +98,18 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             GetMocks(controller).PresenterSession.Verify(Sub(p) p.SelectNextItem(), Times.Once)
         End Sub
 
+        <WorkItem(1181, "https://github.com/dotnet/roslyn/issues/1181")>
+        <Fact>
+        Public Sub UpAndDownKeysShouldStillNavigateWhenDuplicateItemsAreFiltered()
+            Dim item = CreateItems(1).Single()
+            Dim controller = CreateController(items:={item, item}, waitForPresentation:=True)
```

Comment:
Final update: in the case of `items:=CreateItems(2)` where both items have the same contents but aren't considered equal (or equivalent), the arrow keys won't appear to get swallowed so this is a non-issue.  Leaving the test with `items:={item, item}` as it correctly verifies that duplicate items (in this case `ReferenceEquals()`, in the manual repro case `bool Equals(object)`) are appropriately removed.